notifier/notifier_test.go

```diff
@@ -322,7 +330,13 @@ func TestHandlerQueuing(t *testing.T) {
 		select {
 		case expected := <-expectedc:
 			var alerts []*Alert
-			err := json.NewDecoder(r.Body).Decode(&alerts)
+
```

Comment:
where do we close the r.Body?

Reply:
In the case of an HTTP request, you don't need to close the body, only when it's a response.

See https://golang.org/pkg/net/http/#Request:

```
    // The Server will close the request body. The ServeHTTP
    // Handler does not need to.
```